src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = days * dailyCost;
+
+  if (days >= 7) {
+    totalCost = totalCost - 50;
+  } else if (days >= 3) {
```

Comment:
4. [CHECKLIST] Prefer if with return over if else to simplify later conditions.

Same with the line number 11, use `return` after `if` inside a function.